cmd/swarm/secondary.go

```diff
@@ -0,0 +1,71 @@
+package swarm
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+const (
+	DefaultProject = "OHSS"
+)
+
+var (
+	products = []string{"\"Openshift Dedicated\"", "\"Openshift Online Pro\"", "\"OpenShift Online Starter\"", "\"Red Hat OpenShift Service on AWS\"", "\"HyperShift Preview\""}
+)
+
+var secondaryCmd = &cobra.Command{
+	Use:   "secondary",
+	Short: "List unassigned JIRA issues based on criteria",
+	Long: `Lists unassigned Jira issues from the 'OHSS' project
+		for the following Products
+		- OpenShift Dedicated
+		- Openshift Online Pro
+		- OpenShift Online Starter
+		- Red Hat OpenShift Service on AWS
+		- HyperShift Preview
+		- Empty 'Products' field in Jira
+		with the 'Summary' field  of the new ticket not matching the following
+		- Compliance Alert
+		and the 'Work Type' is not one of the RFE or Change Request `,
+	Example: `#Collect tickets for secondary swarm
+		osdctl swarm secondary`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		jiraClient, err := utils.GetJiraClient()
+		if err != nil {
+			return fmt.Errorf("failed to get Jira client: %w", err)
+		}
+		// Build JQL query
+		jql := buildJQL()
+		// Search jira issues
+		issues, _, err := jiraClient.Issue.Search(jql, nil)
```

Comment:
@devppratik : 
The constructed query within the jira ui does not give the following error.  Yet, this command gives the error

` Error: error fetching JIRA issues: The value 'OHSS' does not exist for the field 'project'.: request failed. Please analyze the request body for more details. Status code: 400`